core/workspace.js

```diff
@@ -189,6 +194,54 @@ Blockly.Workspace.prototype.getTopBlocks = function(ordered) {
   return blocks;
 };
 
+/** Add a block to the list of blocks keyed by type.
```

Comment:
First line of a JSDoc should be on the next line (/** should be on its own).  Here and method below.